Formula/mmseqs2.rb

```diff
@@ -59,3 +60,19 @@ def caveats
     assert_predicate (testpath/"res.tsv").size, :positive?
   end
 end
+
+__END__
+diff --git a/lib/simde/simde/simde-common.h b/lib/simde/simde/simde-common.h
+index ee85bfac..1d8038ab 100644
+--- a/lib/simde/simde/simde-common.h
++++ b/lib/simde/simde/simde-common.h
+@@ -166,7 +166,7 @@
+
+ #if !defined(SIMDE_NO_CHECK_IMMEDIATE_CONSTANT)
+   #if defined(SIMDE_CHECK_CONSTANT_) && SIMDE_DETECT_CLANG_VERSION_NOT(9,0,0)
+-    #define SIMDE_REQUIRE_CONSTANT(arg) HEDLEY_REQUIRE_MSG(SIMDE_CHECK_CONSTANT_(arg), "`" #arg "' must be constant")
++    #define SIMDE_REQUIRE_CONSTANT(arg)
+   #else
+     #define SIMDE_REQUIRE_CONSTANT(arg)
+   #endif
+
```

Comment:
Is this also submitted upstream?

Reply:
Now it is: https://github.com/soedinglab/MMseqs2/commit/9d3ab794fb84c4044a95672805e994e135318fcf

Reply:
Could you use the patch from upstream then? For reference: https://docs.brew.sh/Formula-Cookbook#patches

Reply:
I replaced the patch and added a comment